### PR TITLE
Happy Number

### PR DESCRIPTION
**Pull request by ifenil**
# Pull Request Template

## Description

A happy number is a number Starting with any positive integer, replace the number by the sum of the squares of its digits, and repeat the process until the number equals 1 if we get 1 so that number is Happy else not Happy.

## Have you made changes in [README](https://github.com/codedecks-in/LeetCode-Solutions/blob/master/README.md) file ?
- [x] Added problem & solution under correct topic.
- [x] Specified Space & Time complexity.
- [x] Specified difficulty level, tag & Note(if any).

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [x] Test A
- [x] Test B

## Make sure all below guidelines are followed else PR will get Reject:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code so that it is easy to understand
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by lokendra1704 — CHANGES_REQUESTED** [review]
In Happy-Number.java, The use of HashSet can be avoided. Can you improve your solution?

**Comment by ifenil** [comment]
can i use list inplace set?

**Comment by ifenil** [comment]
i made changes, now can you merge it

**Review by lokendra1704 — DISMISSED** [review]

**Comment by ifenil** [comment]
I just made changes in readme file

**Review by lokendra1704 — CHANGES_REQUESTED** [review]
You repeated few names. Can you fix it?

**Comment by ifenil** [comment]
Ok I check it

**Comment by lokendra1704** [comment]
> i made changes, now can you merge it

I like the new Solution 👌

File: README.md [hunk 1]
```diff
@@ -357,11 +358,16 @@ DISCLAIMER: This above mentioned resources have affiliate links, which means if
...
 | [Kevin Chittilapilly](https://github.com/KevinChittilapilly) <br> <img src="https://github.com/KevinChittilapilly.png" width="100" height="100"> | India   | Java                 | [Leetcode](https://leetcode.com/being_kevin/) <br> [Hackerrank](https://www.hackerrank.com/ckevinvarghese11) <br> [Kaggle](https://www.kaggle.com/kevinchittilapilly) |
 | [Nour Grati](https://github.com/Nour-Grati) <br> <img src="https://github.com/Nour-Grati.png" width="100" height="100">                          | Tunisia | Python               | [Leetcode](https://leetcode.com/nourgrati/) <br> [Hackerrank](https://www.hackerrank.com/noor_grati) <br> [Twitter](https://twitter.com/GratiNour1)  
+| [Avinash Trivedi](https://github.com/trivediavinash)  <br>   <img src="https://github.com/trivediavinash.png" width="100" height="100">          | India   | C++                  | [Leetcode](https://leetcode.com/avi_002/) |
+| [Ishika Goel](https://github.com/ishikagoel5628)  <br>  <img src="https://github.com/Nour-Grati.png" width="100" height="100">                   | India   | C++                  | [Leetcode](https://leetcode.com/ishikagoel5628/) |
```
Comment (lokendra1704) [inline README.md:362]: See this is repeated

File: README.md [hunk 2]
```diff
@@ -357,11 +358,16 @@ DISCLAIMER: This above mentioned resources have affiliate links, which means if
...
+| [Fenil Dobariya](https://github.com/ifenil)  <br>  <img src="https://rb.gy/iascx3" width="100" height="100">                                     | India   | Java                 | [Github](https://github.com/ifenil) |                                
+| [Prashansa Tanwar](https://github.com/prashansatanwar)  <br>  <img src="https://github.com/prashansatanwar.png" width="100" height="100">        | India   | C++                  | [Leetcode](https://leetcode.com/prashansaaa/) |
```
Comment (lokendra1704) [inline README.md:364]: And this too

File: README.md [hunk 3]
```diff
@@ -357,11 +358,16 @@ DISCLAIMER: This above mentioned resources have affiliate links, which means if
...
 | [Kevin Chittilapilly](https://github.com/KevinChittilapilly) <br> <img src="https://github.com/KevinChittilapilly.png" width="100" height="100"> | India   | Java                 | [Leetcode](https://leetcode.com/being_kevin/) <br> [Hackerrank](https://www.hackerrank.com/ckevinvarghese11) <br> [Kaggle](https://www.kaggle.com/kevinchittilapilly) |
 | [Nour Grati](https://github.com/Nour-Grati) <br> <img src="https://github.com/Nour-Grati.png" width="100" height="100">                          | Tunisia | Python               | [Leetcode](https://leetcode.com/nourgrati/) <br> [Hackerrank](https://www.hackerrank.com/noor_grati) <br> [Twitter](https://twitter.com/GratiNour1)  
+| [Avinash Trivedi](https://github.com/trivediavinash)  <br>   <img src="https://github.com/trivediavinash.png" width="100" height="100">          | India   | C++                  | [Leetcode](https://leetcode.com/avi_002/) |
```
Comment (lokendra1704) [inline README.md:361]: and this one too

**Review by lokendra1704 — CHANGES_REQUESTED** [review]

**Review by lokendra1704 — DISMISSED** [review]

**Comment by ifenil** [comment]
Ok I do it

**Comment by ifenil** [comment]
now it's solved can you pls check it

**Review by lokendra1704 — APPROVED** [review]

**Comment by ifenil** [comment]
Why it's not merge yet?

**Comment by lokendra1704** [comment]
> Why it's not merge yet?

Only @GouravRusiya30 can merge the requests. He might be busy. Don't worry, It will get merged soon.

**Pull request merged**